src/System.Net.HttpListener/src/System/Net/Windows/HttpListener.Windows.cs

```diff
@@ -2017,7 +2017,7 @@ internal ChannelBinding GetChannelBindingFromTls(ulong connectionId)
             do
             {
                 blob = new byte[size];
-                fixed (byte* blobPtr = blob)
+                fixed (byte* blobPtr = &blob[0])
```

Comment:
Above we assert that size >= 0; if it could actually be 0, this would fail.  But that assert is odd, as on the line immediately before it, it's being set to s_requestChannelBindStatusSize + 128, so unless s_requestChannelBindStatusSize could be negative or so large that it causes an overflow, size could never be 0.  Seems like that assert should be changed to be size > 0 rather than size >= 0.